src/common/components/repository-input/index.js

```diff
@@ -48,6 +50,24 @@ export class RepositoryInput extends Component {
     const isTouched = input.inputValue.length > 2;
     const isValid = !!input.repository && !input.error;
 
+    let submitButton;
+    if (conf.get('LP_API_USERNAME')) {
+      // Main path, for use in production.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Creating...' : 'Create' }
+        </Button>
+      );
+    } else {
+      // If the Launchpad API isn't configured, fall back to just verifying
+      // the repository.  This is handy in development.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Verifying...' : 'Verify' }
+        </Button>
+      );
+    }
```

Comment:
@cjwatson: Yep - my comment was more to give you the information should you wish to abort that change. Continuing is also fine. 👍